src/lib/mail/index.ts

```diff
@@ -13,12 +16,11 @@ const mg = mailgun.client({
   key: process.env.MAILGUN_API_KEY || '',
 });
 
-interface Email {
-  to: string;
-  from: string;
-  subject: string;
-  html: string;
-}
+interface Email
+  extends Required<
+      Pick<MailgunMessageData, 'to' | 'from' | 'subject' | 'html'>
```

Comment:
I can revert this change if you want. It just felt more Typescript-y to me, but it's ultimately not necessary lol.